manifests/r/RevoUninstaller/RevoUninstallerPro/4.4.5/RevoUninstaller.RevoUninstallerPro.locale.en-US.yaml

```
@@ -20,7 +19,7 @@ Description: Revo Uninstaller is a freeware uninstall utility. It has powerful f
 Moniker: revo-pro
 Tags:
 - revo
-- revouninstaller
+- revo uninstaller pro
```

```suggestion
- revo-uninstaller-pro
```
from https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.0.0/defaultLocale.md#optional-field-10